corehq/apps/users/views/__init__.py

```diff
@@ -422,18 +422,24 @@ def _fmt_result(domain, u):
 
     @property
     @memoized
+    def can_restrict_access_by_location(self):
+        return self.domain_object.has_privilege(privileges.RESTRICT_ACCESS_BY_LOCATION)
+
+    @property
+    @memoized
     def user_roles(self):
         user_roles = [AdminUserRole(domain=self.domain)]
         user_roles.extend(sorted(
             UserRole.by_domain(self.domain),
             key=lambda role: role.name if role.name else u'\uFFFF'
         ))
 
-        #  indicate if a role has assigned users, skip admin role
-        for i in range(1, len(user_roles)):
-            role = user_roles[i]
-            role.__setattr__('hasUsersAssigned',
-                             True if len(role.ids_of_assigned_users) > 0 else False)
+        for role in user_roles[1:]:
```

Comment:
Nit: This is kind of a "magic number". Can you either comment why this is `[1:]` or add something in the loop to show that we are skipping the `AdminUserRole`.